src/ast/ddl.rs

```diff
@@ -2524,3 +2524,31 @@ impl fmt::Display for CreateConnector {
         Ok(())
     }
 }
+
+/// `RenameTableNameKind` is the kind used in `ALTER TABLE _ RENAME TO/AS _`.
+/// Used in MySQL.
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub enum RenameTableNameKind {
+    As(ObjectName),
```

Comment:
Can we add a link to the mysql docs, flagging that its the one that supports the `AS` syntax?

Reply:
Done